src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/MarshalTests.cs

```diff
@@ -298,10 +298,10 @@ public static void GetComObjectData()
         [Fact]
         public static void GetHINSTANCE()
         {
-             Assert.Throws<ArgumentNullException>(() => Marshal.GetHINSTANCE(null));
-             IntPtr ptr = Marshal.GetHINSTANCE(typeof(int).Module);
-             IntPtr ptr2 = Marshal.GetHINSTANCE(typeof(string).Module);
-             Assert.Equal(ptr, ptr2);
+            AssertExtensions.Throws<ArgumentNullException>("m", () => Marshal.GetHINSTANCE(null));
```

Comment:
Maybe we should just bulk edit Assert.Throws in the tree to be AssertExtensions..